concepts/pointers/about.md

```diff
@@ -229,7 +233,7 @@ int main() {
 It may not be obvious, but the array of strings is declared and initialized as a pointer to an array of `char` arrays.
 When the array of `char` arrays is passed into `print_strings()`, the signature describes it as a pointer to pointer(s) to `char`.
 
-So how do we know that `char **` is really a pointer to an array of pointers to char arrays, and not a pointer to a pointer of a single char?
+So how do we know that `char **` is really a pointer to an array of pointers to `char` arrays, and not a pointer to a pointer of a single `char`?
 We don't.
 This is one of the may things that can make using pointers confusing.
```

Comment:
Maybe you intended these to render on new lines, as in other cases of this style of prose?
If so, you'll need a couple extra new lines else the markdown is still rendered on all one line IIRC.
```suggestion
So how do we know that `char **` is really a pointer to an array of pointers to `char` arrays, and not a pointer to a pointer of a single `char`?

We don't.

This is one of the may things that can make using pointers confusing.
```